src/_pytest/tracemalloc.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import annotations
+
+
+def tracemalloc_message(source: object) -> str:
+    if source is None:
+        return ""
+
+    try:
+        import tracemalloc
+    except ImportError:
+        return ""
```

Comment:
this should be covered on pypy

Reply:
Leave that comment in the code?  More likely for future maintainers to see it that way 🙂

Reply:
Ah what I mean to say is that the coverage check is failing for this branch, but it shouldn't and I don't know why